views/products/demo/at2.tpl

```diff
@@ -0,0 +1,15 @@
+<p>
+  The
+  <a href="https://factory.c4dt.org/incubator/at2/demo/">AT2 Demonstrator</a>
+  will show you the power lying behind this technology. In a tutorial approach,
+  you'll be taken through the creation of your account on the deployed test
+  network. Then, you'll be able to send some assets to other accounts already
+  registered there, as you would do on other distributed ledgers To show that
+  indeed AT2 is blazingly fast, the last part is a speed test where you can send
+  as many transactions as you want and measure how well the network is handling
+  the load.
+</p>
+
+<a href="https://factory.c4dt.org/incubator/stainless/demo/" class="tab-link">
```

Comment:
```suggestion
<a href="https://factory.c4dt.org/incubator/at2/demo/" class="tab-link">
```
;-)

Reply:
Damn, you found my source of inspiration :sweat_smile: